apps/aeutils/test/data/epoch_full.yaml

```diff
@@ -138,8 +138,12 @@ chain:
     db_path: .
     # Hard forks
     hard_forks:
-      "1": 0
-      "2": 1
+      "1":
+        height: 0
+        use_hardcoded_directory : true
```

Comment:
Why this explicit option is needed? What happens if directory is not provided, couldn't be that used to default to the hardcoded dirs?

Reply:
That's possible but if you have custom blockchain, for example a hyperchain, you would either need to avoid using the same protocol versions as the mainnet or provide an empty directory. mainnet hasn't checked for account or contract files since Lima.